libs/wingsdk/src/cloud/function.ts

```diff
@@ -68,6 +68,8 @@ export class Function extends Resource implements IInflightHost {
 
     super(scope, id);
 
+    this.handler = handler;
```

Comment:
Why are we setting `this.handler` twice? (here and ~10 lines down again)

Reply:
Whoops! leftover from a merge conflicts, thanks for the catch!